linux_os/guide/system/software/integrity/software-integrity/aide/aide_disable_silentreports/ansible/ubuntu.yml

```diff
@@ -0,0 +1,19 @@
+# platform = multi_platform_ubuntu
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+- name: Ensure aide is installed
```

Comment:
Please, ensure the [Projetc Style Guide](https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#ansible) for Ansible Playbooks.